apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -4,8 +4,30 @@
       <oc-table-cell shrink :colspan="firstColumn ? 2 : 1"></oc-table-cell>
       <oc-table-cell colspan="2">
         <div class="uk-text-meta uk-flex uk-flex-middle">
-          <oc-icon name="repeat" class="uk-preserve-width" />
-          <span class="uk-padding-remove uk-margin-xsmall-left uk-text-truncate files-collaborators-collaborator-reshare-information">{{ $_reshareInformation }}</span>
+          <oc-icon name="repeat" class="uk-preserve-width oc-icon-xsmall" />
+          <a :id="$_resharerToggleId" :aria-label="$gettext('Show resharer details')" class="uk-padding-remove uk-margin-xsmall-left uk-text-truncate files-collaborators-collaborator-reshare-information">{{ $_reshareInformation }}</a>
+          <oc-drop
+            :dropId="$_resharerToggleId + '-drop'"
+            :toggle="'#' + $_resharerToggleId"
+            mode="click"
+            :options="{pos:'bottom', delayHide: 0}"
+            class="uk-width-medium"
+            ref="menu"
+            closeOnClick
+          >
+            <h4><translate>Reshared by:</translate></h4>
+            <ul class="uk-list uk-list-divider uk-overflow-hidden uk-margin-remove">
+              <li v-for="resharer in collaborator.resharers" :key="resharer.name">
+                <div class="uk-flex uk-flex-middle uk-flex-left">
+                  <avatar-image v-if="collaborator.shareType === shareTypes.user" class="uk-margin-small-right" :width="48" :userid="resharer.name" :userName="resharer.displayName" />
```

Comment:
`v-if` part doesn't make sense. We want to show an avatar if the shareType of the **resharer** is user. shareType of the **owner** is irrelevant.

Reply:
resharer object doesn't contain the shareType information so far.

Reply:
right, this was a copy-paste